lark/lexer.pyi

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+
+from typing import Tuple, Iterator, Sized
+from abc import abstractmethod, ABC
+
+
+class Pattern(ABC):
+
+    @abstractmethod
+    def to_regexp(self) -> str:
+        ...
+
+
+class PatternStr(Pattern):
+
+    def to_regexp(self) -> str:
+        ...
+
+
+class PatternRE(Pattern):
+
+    def to_regexp(self) -> str:
+        ...
+
+
+class TerminalDef:
+    name: str
+    pattern: Pattern
+    priority: int
+
+
+class Token(str):
```

Comment:
Missing `Token.value`

Reply:
Docs not mentioned. I thought it was used to represent itself in `str(self)`.

Reply:
It's better to look at the code.